src/app/logic/calculatePoints.js

```diff
@@ -0,0 +1,34 @@
+// eslint-disable-next-line import/extensions
+import { DIFFICULTY_LEVELS } from '../data/consts.js';
+
+export function calculatePoints(
+  correctAnswersScore,
+  isAnswerCorrect,
+  difficultyLevelsProperty,
+) {
+  if (
+    DIFFICULTY_LEVELS[difficultyLevelsProperty].calculatingPoints ===
+    DIFFICULTY_LEVELS.medium.calculatingPoints
+  ) {
+    let newCorrectAnswersScore = correctAnswersScore;
+    if (isAnswerCorrect) {
+      newCorrectAnswersScore = correctAnswersScore + 1;
+    }
+    return newCorrectAnswersScore;
+  }
+  if (
+    DIFFICULTY_LEVELS[difficultyLevelsProperty].calculatingPoints ===
+    DIFFICULTY_LEVELS.hard.calculatingPoints
+  ) {
+    let newCorrectAnswersScore = correctAnswersScore;
+    if (isAnswerCorrect) {
+      newCorrectAnswersScore += 1;
+    } else {
+      newCorrectAnswersScore -= 1;
+    }
+    return newCorrectAnswersScore;
+  }
```

Comment:
Perhaps you could use the ternary operator here also `isAnswerCorrect ? correctAnswerScore + 1 : correctAnswerScore - 1`

Reply:
ESLint doesn't like here ternary operator

Reply:
You would need to assign it to the variable that you want to return also, see the previous reply

Reply:
that works! great!